core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -669,9 +670,10 @@ class AutoTuner(
             latestPluginVersion match {
               case Some(ver) =>
                 if (ToolUtils.compareVersions(jarVer, ver) < 0) {
+                  val jarURL = WebCrawlerUtil.getPluginMvnDownloadLink(ver)
                   appendComment(
-                    "A newer RAPIDS Accelerator for Apache Spark plugin\n" +
-                      s"  jar is available [$ver].\n" +
+                    "A newer RAPIDS Accelerator for Apache Spark plugin is available:\n" +
+                      s"  $jarURL\n" +
                       s"  Current version is $jarVer.")
```

Comment:
Let's change "Current version is..." to "Version used in application is...." just to make sure they understand there is a difference between current/latest version available and what they are running with.

Reply:
> Let's change "Current version is..." to "Version used in application is...." just to make sure they understand there is a difference between current/latest version available and what they are running with.

I included a unit test for that:

the output will include the following comment
```
- Multiple RAPIDS Accelerator for Apache Spark plugin jar
   exist on the classpath.
   Make sure to keep only a single jar
```